pkg/cloudinit/cloudinit.go

```diff
@@ -0,0 +1,63 @@
+package cloudinit
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+
+	harvesterv1beta1 "github.com/harvester/node-manager/pkg/apis/node.harvesterhci.io/v1beta1"
+)
+
+const AnnotationHash = "harvesterhci.io/hash"
+
+var Directory = "/host/oem"
+
+func RequireLocal(cloudinit *harvesterv1beta1.CloudInit) error {
```

Comment:
I thought this function applied cloudinit to local. Could we add some comments for it or make the function name more clear?

Reply:
> I thought this function applied cloudinit to local.

It does 🙂 

> Could we add some comments for it or make the function name more clear?

Yeah I'll add a comment for the next revision